src/utils.spec.tsx

```diff
@@ -244,6 +245,40 @@ describe(__filename, () => {
       );
     });
   });
+
+  describe('makeReviewersURL', () => {
+    it('returns a relative url if reviewersHost is falsey', () => {
+      const host = 'https://example.org';
+      const path = '/foo/';
+      const url = `${host}${path}`;
+
+      expect(makeReviewersURL({ reviewersHost: null, url })).toEqual(path);
+    });
+
+    it('returns a relative url when useInsecureProxy is true', () => {
+      const host = 'https://example.org';
+      const path = '/foo/';
+      const url = `${host}${path}`;
+
+      expect(makeReviewersURL({ url, useInsecureProxy: true })).toEqual(path);
+    });
+
+    it('replaces the host with reviewersHost when useInsecureProxy is false', () => {
+      const host = 'https://example.org';
+      const reviewersHost = 'https://example.com';
+      const path = '/foo/';
+      const url = `${host}${path}`;
+      const expectedUrl = `${reviewersHost}${path}`;
+
+      expect(
+        makeReviewersURL({
+          reviewersHost,
+          url,
+          useInsecureProxy: false,
+        }),
+      ).toEqual(expectedUrl);
+    });
+  });
```

Comment:
Should we also consider the query string?

Reply:
I added a temporary test which did include a querystring, just to make sure it all works. I can add that back in as a permanent test.

Reply:
If the test case passes, sure that'd be useful.